deploy/README.md

```diff
@@ -66,29 +95,34 @@ CDK should be bootstrapped once, if this hasn't been done before.
 cdk bootstrap
 ```
 
-In order to deploy, check that the 
+In order to deploy, check that the
 stack synthesizes correctly and then deploy.
 
 ```sh
-cdk synth
-cdk deploy
+cdk synth --context "env=dev"
```

Comment:
Can the context be inferred instead of having to be explicit? Simpler to write a `cdk synth && cdk deploy`... perhaps in this README we shouldn't "leak" too much of the UMCCR-specific details we need for our own deployment.

Perhaps assuming that the "default" env is the one having the required creds should clear up this step and help 3rd party deployers?

Reply:
Yeah, the default env is the "dev" env, so that extra context isn't required.